Makefile

```diff
@@ -7,13 +7,16 @@ DIST_ROOT=dist
 DIST_FOLDER_NAME=mattermost-load-test
 DIST_PATH=$(DIST_ROOT)/$(DIST_FOLDER_NAME)
 
-
 all: install
 
 install:
+	dep ensure
```

Comment:
The documentation suggested modifying the glide (now dep) files to point at the desired branch, but nothing triggered a reevaluation of the dependencies.